saddle-core/src/test/scala/org/saddle/SeriesCheck.scala

```diff
@@ -501,7 +503,7 @@ class SeriesCheck extends Specification with ScalaCheck {
       Series(Vec(1, 2, 3), Index(10, 20, 30))
         .reindex(Index(10, 11, 12, 13, 20, 21, 22, 23), FillForward, limit=1)
         .values
-        .must_==(Vec[Int](1, 1, na, na, 2, 2, na, na))
+        .must_==(Vec[Int](1, 1, naI, naI, 2, 2, naI, naI))
```

Comment:
No implicit conversion?

Reply:
implicit conversion did not happen, even if naToInt is explicitly imported.

Reply:
Doesn't scala 3 require a specific import to enable implicit conversions?

Reply:
The implicit conversions still work, but implicit function values aren't used in implicit conversions in scala 3. They must be converted into regular methods.
```scala
//     implicit val naToInt: na.type => Int = (_: na.type) => scalar.ScalarTagInt.missing
implicit def naToInt(@unused it: na.type): Int = scalar.ScalarTagInt.missing
```

Reply:
If backward compat must be preserved, this could be done:
```scala
// remove implicit modifier of the function value
val naToInt: na.type => Int = (_: na.type) => scalar.ScalarTagInt.missing
implicit def naToIntDef(@unused it: na.type): Int = scalar.ScalarTagInt.missing
```

Reply:
Reference https://docs.scala-lang.org/scala3/reference/changed-features/implicit-conversions-spec.html#differences-with-scala-2-implicit-conversions

Reply:
Addressed in 58b38da.